thriftpy2/contrib/aio/rpc.py

```diff
@@ -56,18 +59,22 @@ def make_server(service, handler,
                 proto_factory=TAsyncBinaryProtocolFactory(),
                 trans_factory=TAsyncBufferedTransportFactory(),
                 client_timeout=3000, certfile=None,
-                keyfile=None, ssl_context=None, loop=None):
+                keyfile=None, ssl_context=None, loop=None,
+                socket_family=socket.AF_INET):
     processor = TAsyncProcessor(service, handler)
 
     if unix_socket:
-        server_socket = TAsyncServerSocket(unix_socket=unix_socket)
+        server_socket = TAsyncServerSocket(
+            unix_socket=unix_socket,
+            socket_family=socket_family)
```

Comment:
If I remember correctly, `AF_INET` is the same level as `AF_UNIX`, which means we shouldn't specify the `socket_family` to `AF_INET` for unix domain socket.

Reply:
Thanks, updated.